### PR TITLE
New cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
New cal csvs including 2 NUTNR post-deploy cal csvs from Irm-10 cruise (recovery of SUMO-9)

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PYV Instrument Cal CSVs [DT-184]](https://trello.com/c/Eo546k32/184-pyv-instrument-cal-csvs-dt-184)

File: calibration/NUTNRB/CGINS-NUTNRB-01088__20230915.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
...73.7, 274.52, 275.34, 276.16, 276.99, 277.81, 278.63, 279.45, 280.27, 281.1, 281.92, 282.74, 283.56, 284.38, 285.21, 286.03, 286.85, 287.67, 288.49, 289.31, 290.14, 290.96, 291.78, 292.6, 293.42, 294.24, 295.07, 295.89, 296.71, 297.53, 298.35, 299.17, 299.99, 300.81, 301.63, 302.46, 303.28, 304.1, 304.92, 305.74, 306.56, 307.38, 308.2, 309.02, 309.84, 310.66, 311.48, 312.3, 313.12, 313.94, 314.76, 315.58, 316.4, 317.22, 318.04, 318.86, 319.68, 320.5, 321.32, 322.13, 322.95, 323.77, 324.59, 325.41, 326.23, 327.04, 327.86, 328.68, 329.5, 330.32, 331.13, 331.95, 332.77, 333.58, 334.4, 335.22, 336.04, 336.85, 337.67, 338.48, 339.3, 340.12, 340.93, 341.75, 342.56, 343.38, 344.19, 345.01, 345.82, 346.64, 347.45, 348.27, 349.08, 349.89, 350.71, 351.52, 352.34, 353.15, 353.96, 354.77, 355.59, 356.4, 357.21, 358.02, 358.84, 359.65, 360.46, 361.27, 362.08, 362.89, 363.7, 364.51, 365.32, 366.13, 366.94, 367.75, 368.56, 369.37, 370.18, 370.99, 371.8, 372.61, 373.41, 374.22, 375.03, 375.84, 376.64, 377.45, 378.26, 379.06, 379.87, 380.67, 381.48, 382.29, 383.09, 383.89, 384.7, 385.5, 386.31, 387.11, 387.91, 388.72, 389.52, 390.32, 391.13, 391.93, 392.73, 393.53, 394.33, 395.13, 395.93, 396.73]",
```
Comment (cooleyky) [inline calibration/NUTNRB/CGINS-NUTNRB-01088__20230915.csv:8]: The values that I spot-checked looked good

File: calibration/NUTNRB/CGINS-NUTNRB-01089__20230914.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
...273.48, 274.28, 275.08, 275.88, 276.68, 277.48, 278.28, 279.08, 279.88, 280.69, 281.49, 282.29, 283.09, 283.89, 284.69, 285.49, 286.29, 287.09, 287.89, 288.69, 289.49, 290.29, 291.09, 291.89, 292.69, 293.49, 294.29, 295.09, 295.89, 296.69, 297.49, 298.29, 299.09, 299.89, 300.69, 301.49, 302.29, 303.09, 303.89, 304.68, 305.48, 306.28, 307.08, 307.88, 308.68, 309.48, 310.28, 311.08, 311.88, 312.68, 313.47, 314.27, 315.07, 315.87, 316.67, 317.47, 318.27, 319.06, 319.86, 320.66, 321.46, 322.26, 323.05, 323.85, 324.65, 325.45, 326.24, 327.04, 327.84, 328.63, 329.43, 330.23, 331.03, 331.82, 332.62, 333.41, 334.21, 335.01, 335.8, 336.6, 337.39, 338.19, 338.99, 339.78, 340.58, 341.37, 342.17, 342.96, 343.76, 344.55, 345.34, 346.14, 346.93, 347.73, 348.52, 349.31, 350.11, 350.9, 351.69, 352.49, 353.28, 354.07, 354.87, 355.66, 356.45, 357.24, 358.03, 358.83, 359.62, 360.41, 361.2, 361.99, 362.78, 363.57, 364.36, 365.15, 365.94, 366.73, 367.52, 368.31, 369.1, 369.89, 370.68, 371.47, 372.26, 373.04, 373.83, 374.62, 375.41, 376.2, 376.98, 377.77, 378.56, 379.34, 380.13, 380.91, 381.7, 382.49, 383.27, 384.06, 384.84, 385.63, 386.41, 387.2, 387.98, 388.76, 389.55, 390.33, 391.11, 391.9, 392.68]",
```
Comment (cooleyky) [inline calibration/NUTNRB/CGINS-NUTNRB-01089__20230914.csv:8]: Spot-checked a few values, and it looks good

**Review by cooleyky — APPROVED** [review]
Looks good

File: calibration/ADCPSL/CGINS-ADCPSL-18261__20230811.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+18261,CC_scale_factor1,0.45,constant; source_file ADCPS-L_WHLS75-1500-I_SN_18261_Inspection_Report_2023-08-11.pdf > Final Report date
+18261,CC_scale_factor2,0.45,constant
+18261,CC_scale_factor3,0.45,constant
+18261,CC_scale_factor4,0.45,constant
+18261,CC_bin_size,0,in cm; return from vendor - not allocated yet
+18261,CC_dist_first_bin,0,in cm; return from vendor - not allocated yet
+18261,CC_orientation,1,1 is upward looking; 0 is downward looking
```
[the remark this answers is not in the document]
Comment (sauuyer) [reply calibration/ADCPSL/CGINS-ADCPSL-18261__20230811.csv:8]: Not sure if "> Final Report date" should be in this file's line 2, or if it needs to be added to ADCPSL-18445's line 2

File: calibration/ADCPSL/CGINS-ADCPSL-18445__20230822.csv [hunk 4]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+18445,CC_scale_factor1,0.45,constant; source_file ADCPS-L_WHLS75-1500-I_SN_18445_Configuration_Summary_2023-08-22.pdf
+18445,CC_scale_factor2,0.45,constant
+18445,CC_scale_factor3,0.45,constant
+18445,CC_scale_factor4,0.45,constant
+18445,CC_bin_size,0,in cm; return from vendor - not allocated yet
+18445,CC_dist_first_bin,0,in cm; return from vendor - not allocated yet
+18445,CC_orientation,1,1 is upward looking; 0 is downward looking
```
[the remark this answers is not in the document]
Comment (sauuyer) [reply calibration/ADCPSL/CGINS-ADCPSL-18445__20230822.csv:8]: Looks good

File: calibration/ADCPSL/CGINS-ADCPSL-18261__20230811.csv (hunk 3 above)
[the remark this answers is not in the document]
Comment (sauuyer) [reply calibration/ADCPSL/CGINS-ADCPSL-18261__20230811.csv:8]: Looks good

**Review by sauuyer — APPROVED** [review]
all changes look good

**Pull request merged**